src-docs/src/views/form_layouts/form_layouts_example.js

```diff
@@ -130,6 +133,31 @@ export const FormLayoutsExample = {
 >
   <EuiRange fullWidth />
 </EuiFormRow>`,
+    },
+    {
+      title: 'Global Full-width',
```

Comment:
If you've got opinions about better docs/example text please feel free to re-write anything 😅 Just wanted to make sure we had an example of this actually working in several of the components.

Reply:
Looks good to me! I think my only comment would be to make the title have sentence casing, per our copy guidelines
```suggestion
      title: 'Global full-width',
```

After looking at it a bit more closely, I'm not totally sure if we should just combine the Full-width and Global full-width examples into 1 single example though - @miukimiu, any thoughts here? We can definitely address that after this PR merges though TBH